config/config.exs

```diff
@@ -49,4 +41,11 @@ config :fields, Fields.AES,
 config :fields, Fields,
   secret_key_base: System.get_env("SECRET_KEY_BASE")
 
+# Google application configuration
+config :elixir_auth_google,
```

Comment:
seems like we can remove this now? https://github.com/dwyl/elixir-auth-google/pull/24